src/components/ProjectForm/ProjectForm.js

```diff
@@ -0,0 +1,163 @@
+import React, { useState, useEffect } from 'react';
+import CreatableSelect from 'react-select/creatable';
+import { Form, Label, Input, TextArea } from './styles';
+import getAllAndis from '../../api/handlers/andis';
+import getAllTechnologies from '../../api/handlers/technologies';
+import getAllClients from '../../api/handlers/clients';
+
+const ProjectForm = () => {
+
+    const fetchAndis = async () => {
+        const response = await getAllAndis();
+
+        response.data.res.forEach(element => {
+          let andiValues = { value: element["andiid"], label: element["firstname"] };
+          andiOptions.push(andiValues);
+        });
+    }
+
+    const fetchTechnologies = async () => {
+        const response = await getAllTechnologies();
+
+        response.data.res.forEach(element => {
+          let technologyValues = { value: element["technologyid"], label: element["name"] };
+          technologyOptions.push(technologyValues);
+        });
+    }
+
+    const fetchClients = async () => {
+        const response = await getAllClients();
+        response.data.res.forEach(element => {
+          let clientsValues = { value: element["clientid"], label: element["clientname"] };
+          clientOptions.push(clientsValues);
+        });
+    }
+
+    if (andis === undefined) {
+      fetchAndis();
+    }
+
+    if (technologies === undefined) {
+      fetchTechnologies();
+    }
+
+    if (technologies === undefined) {
+      fetchClients();
+    }
+
+    const clientOptions = [
```

Comment:
The [useState](https://reactjs.org/docs/hooks-state.html) hook is perfect for this, rather than defining empty lists you can use the state to store them as default values, then update these using the setter